### PR TITLE
Instrument website with AEM Experimentation plugin

### PR DESCRIPTION
**Pull request by raji-unni**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--cn-website--netcentric.hlx.live/
- After: https://aem-experimentation--cn-website--netcentric.hlx.page/drafts/experiments/what-we-do

### CODE REVIEW

**Comment by ramboz** [comment]
@raji-unni To address the linting issues, we can add `plugins/` to the `.gitignore` file at the root of the project

File: scripts/lib-franklin.js [hunk 1]
```diff
@@ -463,41 +466,65 @@ export function buildBlock(blockName, content) {
...
-      });
-      await Promise.all([cssLoaded, decorationComplete]);
+      await loadModule(blockName, jsPath, cssPath, block);
     } catch (error) {
       // eslint-disable-next-line no-console
       console.log(`failed to load block ${blockName}`, error);
     }
-    block.setAttribute('data-block-status', 'loaded');
-    const event = new CustomEvent('block-loaded', { detail: { block, blockName } });
-    document.body.dispatchEvent(event);
```
[the remark this answers is not in the document]
Comment (raji-unni) [reply scripts/lib-franklin.js]: Added back these 2 lines of code

File: scripts/scripts.js [hunk 2]
```diff
@@ -14,9 +14,43 @@ import {
...
+      return res;
+    }, {});
+}
+
+// Define an execution context
+const pluginContext = {
+  getAllMetadata,
+  getMetadata,
+  loadCSS,
+  // loadScript,
```
[the remark this answers is not in the document]
Comment (raji-unni) [reply scripts/scripts.js:48]: When I comments this out, the page http://localhost:3000/drafts/experiments/what-we-do does not load with console error -
Uncaught ReferenceError: loadScript is not defined
    at scripts.js:48:3

[the remark this answers is not in the document]
Comment (ramboz) [reply scripts/scripts.js:48]: Right, I'm not saying you should uncomment the line. I'm saying to completely remove it. `loadScript` is not defined and also not used in the plugin, so we might as well remove the reference to it (even if commented out)

[the remark this answers is not in the document]
Comment (raji-unni) [reply scripts/scripts.js]: Sure got it. I have removed the line.

File: plugins/experimentation/.eslintignore
Comment (nc-andreashaller) [inline plugins/experimentation/.eslintignore:1]: is this file needed if we have the ignore in the root file already?

**Review by nc-andreashaller — APPROVED** [review]

File: plugins/experimentation/.eslintignore
Comment (ramboz) [reply plugins/experimentation/.eslintignore:1]: the way the plugins work is that they essentially cache all the files from the source repo: https://github.com/adobe/aem-experimentation

it's essentially creating a subtree or submodule it git. in theory you could remove some of the files, but it's easier to keep them as-is so fetching updates is easier and doesn't cause conflicts

**Comment by ramboz** [comment]
@nc-andreashaller Do you want us to go ahead and merge this or are you taking over and merging when you are ready to move forward?
